jdi-light-material-ui-tests/src/main/java/io/github/com/pages/layout/GridFrame.java

```diff
@@ -0,0 +1,18 @@
+package io.github.com.pages.layout;
```

Comment:
To use folders *.layouts for all classes as it will be the right folder after pull request https://github.com/jdi-testing/jdi-light/pull/2841/ is merged

Reply:
Please update folders for all possible classes

Reply:
Finally it was decided to do as in material ui site - "layout" folder. PR #2841 will be updated
Please update this PR